### DISCUSSION TITLE
Release 0.3?

### DISCUSSION THREAD

**Discussion by tramhao** [Serial-ATA/lofty-rs#221] [Q&A] [answered]
I'm currently working on a side project which is a music player(termusic) written in rust. I'm using the 0.3 version of lofty from master branch. However, this can only work locally as 0.3 is not on crates.io yet.

Would you please publish it? Thanks.

**Comment by Serial-ATA** [comment]
I had planned to release 0.3 awhile ago, but kept finding issues and adding features. The published versions currently only read a single tag from the files. I've been working in the [multi-tag-files](/Serial-ATA/lofty-rs/tree/multi-tag-files) branch for awhile now, as it required completely rewriting the crate. I'm nearly done with it and that will be released under 0.3.

**Comment by tramhao** [comment]
Thanks for the update and I'll keep watching for your progress. It's a great work and simplified things a lot.

**Comment by leo60228** [comment]
I'd like to migrate a project of mine to Lofty, but I don't want to have to rewrite it a few weeks later. Should I use Lofty 0.2, git, or wait for 0.3?

**Comment by Serial-ATA** [comment]
@leo60228 I'd recommend waiting for 0.3. 0.2 and the `main` branch have a fair amount of issues I've addressed in the `multi-tag-files` branch. I should finally have some time in the coming weeks to push it over the finish line.

**Comment by tramhao** [comment]
It seems works fine now. Would you please publish a version first? I cannot publish as the dependency is not published yet. For some bugs, later we can fix when we find it or user report it. Thanks.

**Comment by Serial-ATA** [comment]
Yeah, I was planning on publishing. I just need to look at the docs to make sure I haven't left anything behind from previous versions. If all goes well it'll be out later today or tomorrow.

**Comment by tramhao** [comment]
It'll be a great help! I'm currently using id3, metaflac, mp4ameta, lewton, ogg, and even write some nasty code for ogg pictures. With lofty, all things about metadata will be solved!

**Comment by Serial-ATA** [comment, marked as answer]
0.3 is out 🎉
